java/example/src/main/java/io/vitess/example/VitessClientExample.java

```diff
@@ -52,9 +52,9 @@ public static void main(String[] args) {
     try (RpcClient client = new GrpcClientFactory()
         .create(ctx, args[0]); VTGateBlockingConnection conn = new VTGateBlockingConnection(
         client)) {
-      VTSession session = new VTSession("@master", ExecuteOptions.getDefaultInstance());
+      VTSession session = new VTSession("@primary", ExecuteOptions.getDefaultInstance());
```

Comment:
Actually I had left some of these references to `@master` to confirm backward compatibility. But since those PR tests passed, guess it is fine to change these now.